developers/advanced/ingestion-level-transformations.md

```diff
@@ -319,7 +319,7 @@ We can apply one transformation to extract the `userId` and then another one to
 "ingestionConfig": {
     "transformConfigs": [
       {
-        "columnName": "userOid",
+        "columnName": "userId",
```

Comment:
I don't think this is a typo. It is used in the next transform

Reply:
Yes, it seems a misunderstanding occurred, abandoning the diff. In the future helpfully a clairifcation could be added for the user0id column being an intermediate result for chained functions.